src/MartinGeorgiev/Doctrine/DBAL/Types/Ltree.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MartinGeorgiev\Doctrine\DBAL\Types;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use MartinGeorgiev\Doctrine\DBAL\Types\Exceptions\InvalidLtreeForDatabaseException;
+use MartinGeorgiev\Doctrine\DBAL\Types\Exceptions\InvalidLtreeForPHPException;
+use MartinGeorgiev\Doctrine\DBAL\Types\ValueObject\Ltree as LtreeValueObject;
+
+final class Ltree extends BaseType
```

Comment:
```suggestion
/**
 * Implementation of PostgreSQL LTREE data type.
 *
 * @see https://www.postgresql.org/docs/13/ltree.html
 * @since 3.5
 *
 * @author Pierre-Yves Landuré
 */
final class Ltree extends BaseType
```